app/src/main/java/com/trpapps/helloworldapp/BeerXmlParser.java

```diff
@@ -0,0 +1,114 @@
+package com.trpapps.helloworldapp;
+
+/**
+ * Created by prudnick on 7/2/2015.
+ */
+
+import java.io.InputStream;
+import java.io.IOException;
+import java.util.List;
+import java.util.ArrayList;
+
+import org.xmlpull.v1.XmlPullParserException;
+import org.xmlpull.v1.XmlPullParser;
+
+import android.util.Xml;
+
+import com.trpapps.helloworldapp.Beer;
+
+public class BeerXmlParser {
+
+    private Beer readBeer(XmlPullParser parser) throws XmlPullParserException, IOException {
+        parser.require(XmlPullParser.START_TAG, ns, "beer");
+        String name = null;
+        String alc = null;
+        while (parser.next() != XmlPullParser.END_TAG) {
```

Comment:
NIE WIERZĘ że to musi być takie skomplikowane :D w pythonie to by było z 10 linijek